models/src/main/java/com/vimeo/networking2/TeamBranding.kt

```diff
@@ -0,0 +1,57 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import com.vimeo.networking2.common.Entity
+
+/**
+ * Stores user customization related information about a team.
+ */
+@JsonClass(generateAdapter = true)
+data class TeamBranding(
+
+    /**
+     * The accent color of the team.
```

Comment:
We should describe the format that the color is returned in so that the client will know how to parse it.